nuto/mechanics/cell/Jacobian.h

```diff
@@ -54,43 +53,43 @@ class Jacobian
             {
             case 2:
             {
-                Eigen::Matrix<double, 2, 1> jacobian = CalculateFixedSize<2,1>(nodeValues, derivativeShapeFunctions);
-                Eigen::Matrix<double, 2,2> extendedJacobian;
-                mNormal = Eigen::Vector2d(jacobian(1,0),-jacobian(0,0)).normalized();
+                Eigen::Matrix<double, 2, 1> jacobian = CalculateFixedSize<2, 1>(nodeValues, derivativeShapeFunctions);
+                Eigen::Matrix<double, 2, 2> extendedJacobian;
+                mNormal = Eigen::Vector2d(jacobian(1, 0), -jacobian(0, 0)).normalized();
                 extendedJacobian.col(0) = jacobian;
                 extendedJacobian.col(1) = mNormal;
                 mJacobian = jacobian;
-                mInvJacobian = extendedJacobian.inverse().block<1,2>(0,0);
+                mInvJacobian = extendedJacobian.inverse().block<1, 2>(0, 0);
                 mDetJacobian = -extendedJacobian.determinant();
                 break;
             }
             case 3:
             {
-                Eigen::Matrix<double, 3, 2> jacobian = CalculateFixedSize<3,2>(nodeValues, derivativeShapeFunctions);
-                Eigen::Matrix<double, 3,3> extendedJacobian;
+                Eigen::Matrix<double, 3, 2> jacobian = CalculateFixedSize<3, 2>(nodeValues, derivativeShapeFunctions);
+                Eigen::Matrix<double, 3, 3> extendedJacobian;
                 mNormal = (jacobian.col(0)).cross(jacobian.col(1)).normalized();
                 extendedJacobian.col(0) = jacobian.col(0);
                 extendedJacobian.col(1) = jacobian.col(1);
                 extendedJacobian.col(2) = mNormal;
                 mJacobian = jacobian;
-                mInvJacobian = extendedJacobian.inverse().block<2,3>(0,0);;
+                mInvJacobian = extendedJacobian.inverse().block<2, 3>(0, 0);
+                ;
                 mDetJacobian = extendedJacobian.determinant();
                 break;
-
             }
             default:
             {
                 throw;
             }
             }
-        } else
+        }
+        else
         {
             throw;
```

Comment:
What is going on here? Why don't these give at least some indication what's wrong? I know this was not part of the PR, I just noticed this now.

Reply:
Will be addressed in another PR.